man/bundle-config.ronn

```diff
@@ -272,6 +272,18 @@ For example, to use a mirror of rubygems.org hosted at
 
     bundle config mirror.http://rubygems.org http://rubygems-mirror.org
 
+Bundler allows to set fallback timeout. When it pass Bundler will use original
```

Comment:
Perhaps a wording like:

> Bundler allows setting a fallback timeout. When the timeout is reached, Bundler will use the original source instead of the configured mirror.